core/Dispatcher.hpp

```diff
@@ -70,6 +70,8 @@ namespace olympia
         // Reset the bandwidth
         void reset() { num_can_dispatch_ = 1; }
 
+        uint32_t getCredits() {return unit_credits_;}
```

Comment:
Can be a `const` method.

Reply:
Resolved in [6b0e363](https://github.com/riscv-software-src/riscv-perf-model/pull/140/commits/6b0e3633c1b7331170afda8920f596f9fd16b233)